src/test/java/uk/gov/hmcts/reform/cwrdapi/service/ExcelAdaptorServiceImplTest.java

```diff
@@ -102,4 +113,32 @@ public void parseServiceRoleMappingXlsx() throws IOException {
         assertThat(serviceRoleMapping.getSerivceId()).isEqualTo("BBA9");
 
     }
+
+    @Test
+    public void sendXlsxWithIncorrectHeaders() throws IOException {
+        Workbook workbook = WorkbookFactory
+                .create(new File("src/test/resources/CaseWorkerUserXlsWithInvalidHeaders.xls"));
+
+        Assertions.assertThatThrownBy(() -> excelAdaptorServiceImpl.parseExcel(workbook, CaseWorkerProfile.class))
+                .isExactlyInstanceOf(ExcelValidationException.class)
+                .hasMessage(FILE_MISSING_HEADERS);
+    }
+
+    /*@Test
+    public void sendXlsxWithMissingLocation() throws IOException {
```

Comment:
These test cases will be fixed and uncommented once other dev's code is in place.